smart-contracts/contracts/interfaces/IPublicLock.sol

```diff
@@ -1,5 +1,6 @@
 // SPDX-License-Identifier: MIT
 pragma solidity >=0.5.17 <0.9.0;
+pragma experimental ABIEncoderV2;
```

Comment:
Hum? 🧹

Reply:
Nope, has to keep. It is needed up to 0.8.x to support bytes array in syntax (the `_data` args is one)